src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -2320,7 +2320,7 @@ TEST_F(ClusterTaskManagerTestWithoutCPUsAtHead, ZeroCPUNode) {
 TEST_F(ClusterTaskManagerTest, SchedulingClassCapSpillback) {
   std::shared_ptr<MockWorker> worker =
       std::make_shared<MockWorker>(WorkerID::FromRandom(), 1234);
-  pool_.PushWorker(std::dynamic_pointer_cast<WorkerInterface>(worker));
+  pool_.PushWorker(std::static_pointer_cast<WorkerInterface>(worker));
```

Comment:
The casting is not necessary here, see my example: https://onlinegdb.com/uM2dADv7Q